third_party/nvfuser/version.txt

```diff
@@ -1 +1 @@
-0.0.1
+0.0.2
```

Comment:
Muhahahahaha, now we have to make sure that no one else is bumping the version on other inflight PRs.

Human mutex is on! @kevinstephano